examples/rpl_udp/rpl.c

```diff
@@ -167,7 +167,7 @@ void rpl_udp_dodag(int argc, char **argv)
     printf("Part of Dodag:\n");
     printf("%s\n", ipv6_addr_to_str(addr_str, IPV6_MAX_ADDR_STR_LEN,
                                     (&mydodag->dodag_id)));
-    printf("my rank: %d\n", mydodag->my_rank);
+    printf("my rank: %d\n", NTOHS(mydodag->my_rank.u16));
```

Comment:
But when I remove the member access and the conversion to host byteorder I get:
```
RIOT/examples/rpl_udp/rpl.c:170:5: warning: format '%d' expects argument of type 'int', but argument 2 has type 'network_uint16_t' [-Wformat=]
     printf("my rank: %d\n", (mydodag->my_rank));
     ^
```
On:
`gcc version 4.9.3 20141119 (release) [ARM/embedded-4_9-branch revision 218278]`
for `iot-lab_M3`